themis/build.gradle

```diff
@@ -0,0 +1,2 @@
+configurations.maybeCreate("default")
+artifacts.add("default", file('themis-release.aar'))
```

Comment:
if you have any suggestions how to distributed Themis in a more developers-friendly way, please let me know :)